### ISSUE TITLE
FR: `jj setup` command to prompt user for name/email/settings

### ISSUE THREAD

**Issue by Cretezy** [martinvonz/jj#3597]
**Is your feature request related to a problem? Please describe.**
When setting up jj, everyone must run `jj config set --user user.name ... && jj config set --user user.email ...`.

**Describe the solution you'd like**
A command such as `jj setup` which asks the user for their name, email, and other relevant settings. This command can use the current values set in the config, as well as read from `.gitconfig` for name/email to simplify setup.

This could use https://github.com/mikaelmello/inquire

**Comment by PhilipMetzger** [comment]
I don't think that a command is necessary for that, although it probably should land in `jj init` when we have multiple backends see https://github.com/martinvonz/jj/issues/2747#issuecomment-1927844516.
